utils/lt-code/encoder.ts

```diff
@@ -30,10 +31,20 @@ export class LtEncoder {
       }
     }
 
+    console.log({
+      k: this.k,
+      bytes: this.bytes,
+      checksum: this.checksum,
+      contentType: mapContentType(this.contentType),
```

Comment:
Instead of sending it in the header (which will be present in every block), I think we could send them in the payload. As such information is only useful when the full data is resolved.

I think maybe we could have 

```ts
function appendMetaToBuffer(data: Uint8Array, meta: any): Uint8Array {
   return [JSON.stringify(meta), SEP, data] // somehow merge the data
}
```

and then

```ts
function extraMetaFromBuffer(input: Uinit8Array): [data: Uint8Array, meta: any] {
// ...
}
```

Reply:
Added the implementation in [d016a56](https://github.com/qifi-dev/qrs/pull/4/commits/d016a563320e496456e497c10391c59ef00c6a14)

Reply:
Tried to integrated with your implementation.